spec/lib/core/repository/cms/attribute_builder_spec.rb

```diff
@@ -1,7 +1,87 @@
+# coding: utf-8
 module Core::Repository::CMS
   RSpec.describe AttributeBuilder do
     let(:body) { File.read('spec/fixtures/cms/beginners-guide-to-managing-your-money.json') }
     let(:response) { OpenStruct.new(body: JSON.parse(body)) }
+    let(:ab)  { AttributeBuilder.new(:resource) }
+
+    describe '.set_title_from_label' do
+      let(:hsh) { { 'label' => :foo } }
+
+      it 'copies label values to title' do
+        ab.set_title_from_label(hsh)
+        expect(hsh['title']).to eq hsh['label']
+      end
+    end
+
+    describe '.set_body_from_content_block' do
+      let(:hsh) { { 'blocks' => [{ 'identifier' => 'content', 'content' => :bar }] } }
+
+      it 'uses block_composer to set body' do
+        ab.set_body_from_content_block(hsh)
+        expect(hsh['body']).to eq 'bar'
+      end
+    end
+
+    describe '.translate_attributes_from_raw_blocks' do
+      let(:hsh) do
```

Comment:
Can it be renamed to hash, please? IMHO abbreviation it is only useful when it is very very long word.